tests/model/test_model.py

```diff
@@ -2008,13 +2055,17 @@ def test_notify_users_calling_msg_type(
         vary_each_msg,
         visual_notification_status,
         types_when_notify_called,
+        is_muted_stream: Optional[bool],
+        is_muted_topic: Optional[bool],
```

Comment:
The return value for these functions cannot be None, so using it is confusing.

Consider a real bool value in each case which will strengthen a test case - eg. if mentions work even if a stream or topic are muted, show that. The ids may need adjusting.